pkg/common/common.go

```diff
@@ -1,11 +1,29 @@
 package common
 
 import (
+	"context"
+
+	"github.com/pkg/errors"
+	netv1 "k8s.io/api/networking/v1"
+	"k8s.io/apimachinery/pkg/types"
+	"sigs.k8s.io/controller-runtime/pkg/client"
+
 	loggedcluster "github.com/giantswarm/logging-operator/pkg/logged-cluster"
 )
 
-// ReadUser is the global user for reading logs
-const ReadUser = "read"
+const (
+	// ReadUser is the global user for reading logs
+	ReadUser = "read"
+	// Loki Ingress
+	lokiIngressNamespace = "loki"
+	lokiIngressName      = "loki-gateway"
+	// grafana-agent secret name
+	grafanaAgentResourceName = "grafana-agent-secret"
+)
+
+func GetGrafanaAgentResourceName() string {
```

Comment:
```suggestion
func GrafanaAgentSecretName() string {
```

And why does it need to be in common when it could be here instead https://github.com/giantswarm/logging-operator/blob/508e156efd552cf2004e1cddd12d0c9bb65223e0/pkg/resource/grafana-agent-secret/grafana-agent-secret.go ?

Reply:
because I use it here also
https://github.com/giantswarm/logging-operator/blob/508e156efd552cf2004e1cddd12d0c9bb65223e0/pkg/resource/grafana-agent-config/grafana-agent-config.go#L74

Reply:
Oh I missed that then a renaming is enough :)